src/cookies.test.ts

```diff
@@ -77,7 +109,35 @@ describe('cookies', () => {
 		);
 	});
 
-	it('should be able to get a memoized cookie with days to live and cross subdomain', () => {
+	it('set a session cookie for localhost', () => {
+		Object.defineProperty(document, 'domain', {
+			value: 'localhost',
+		});
+		expect(document.cookie).toEqual('');
+		cookies.setSessionCookie({
+			name: 'cookie-1-name',
+			value: 'cookie-1-value',
+		});
+		expect(document.cookie).toEqual('cookie-1-name=cookie-1-value; path=/');
+	});
+
+	it('does not set a session cookie when the cookie name is invalid', () => {
```

Comment:
could session/non-session tests be merged? not a blocker, no worries if you prefer it this way

Reply:
Good suggestion.

`setSessionCookie` is only used by `libs/getLocale` so we can update this with minimal impact.

 I'll add in another PR so this one remains focussed on the UTC bug fix.